crates/scheduler/Cargo.toml

```diff
@@ -7,7 +7,8 @@ publish = true
 edition = "2021"
 description = "Wasefire scheduler"
 repository = "https://github.com/google/wasefire"
-include = ["/LICENSE", "/build.rs", "/src/"]
+include = ["/LICENSE", "/src/"]
+build = "src/build.rs"
```

Comment:
Did you try `/src/build.rs`? Not sure why `include` has it, but better be explicit if it works.

Reply:
Just tried and it doesn't work. The [documentation](https://doc.rust-lang.org/cargo/reference/manifest.html#the-build-field) is also clear that the file is search at the package root.